src/main/java/forestry/farming/logic/FarmLogicRubber.java

```diff
@@ -152,31 +155,34 @@ private Collection<ICrop> getHarvestBlocks(Vect position) {
         return crops;
     }
 
-    private ArrayList<Vect> processHarvestBlock(Stack<ICrop> crops, Set<Vect> seen, Vect position) {
+    private ArrayList<Vect> processHarvestBlock(Deque<ICrop> crops, Set<IVect> seen, Vect position) {
         World world = getWorld();
 
         ArrayList<Vect> candidates = new ArrayList<>();
 
         // Get additional candidates to return
+        final MutableVect mutable = new MutableVect();
         for (int j = 0; j < 2; j++) {
-            Vect candidate = new Vect(position.x, position.y + j, position.z);
-            if (candidate.equals(position)) {
+            // Vect candidate = new Vect(position.x, position.y + j, position.z);
+            mutable.y = position.y + j;
```

Comment:
X and Z axes of `mutable` are not initialized.
Also please, remove old code instead of commenting it.

Reply:
Ah yes, missed that thanks.